src/main/java/microsoft/exchange/webservices/data/autodiscover/request/AutodiscoverRequest.java

```diff
@@ -534,8 +524,8 @@ protected void writeSoapRequest(URI requestUrl,
    * Write extra headers.
    *
    * @param writer The writer
-   * @throws ServiceXmlSerializationException
-   * @throws javax.xml.stream.XMLStreamException
+   * @throws ServiceXmlSerializationException the service xml serialization exception
+   * @throws XMLStreamException the xML stream exception
```

Comment:
Yes, I'll fix it. It was just a copy-paste from other places (to be in one way). :)